packages/@react-aria/radio/src/utils.ts

```diff
@@ -14,8 +14,8 @@ import {RadioGroupState} from '@react-stately/radio';
 
 interface RadioGroupData {
   name: string,
-  descriptionId: string,
-  errorMessageId: string,
+  descriptionId: string | undefined,
```

Comment:
i believe this is correct as useSlotId might change it to undefined if it can't find the value in the dom